db/seeds.rb

```diff
@@ -187,4 +187,4 @@
   r.feed_format = "video"
 end.save!
 
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+AdminUser.create!(email: 'helios@mojotech.com', password: 'mojojojo', password_confirmation: 'mojojojo') if Rails.env.development?
```

Comment:
if this is only for development, its nice to keep the username/password simpler: admin@example.com/password is pretty standard for development

Reply:
also, lets make sure the readme gets updated